test/Groups/matrixgroups.jl

```diff
@@ -85,6 +85,20 @@
    @test G.mat_iso(xg)==xo
    @test G.mat_iso(GAP.Globals.One(GAP.Globals.GL(3,G.mat_iso.fr.codomain)))==one(G).elm
    @test GAP.Globals.Order(G.mat_iso(diagonal_matrix([z,z,one(F)])))==4
+
+   M = matrix(QQ, [ 0 1 0; -1 0 0; 0 0 -1 ])
+   G = Oscar.isomorphic_group_over_finite_field([ M ])
+   H = GAP.Globals.Group(GAP.julia_to_gap([ 0 1 0; -1 0 0; 0 0 -1 ]))
+   @test GAP.Globals.IsomorphismGroups(G.X, H) != GAP.Globals.fail
```

Comment:
This should work and without warning; since we don't give it generators for the domain/codomain, it will take the default generators, which should align, if I read the code above correctly.
```suggestion
   @test GAP.Globals.GroupHomomorphismByImages(G.X, H) != GAP.Globals.fail
```